frontera/contrib/scrapy/schedulers/frontier.py

```diff
@@ -76,11 +77,14 @@ def __init__(self, crawler):
         self._pending_requests = deque()
         self.redirect_enabled = crawler.settings.get('REDIRECT_ENABLED')
 
-        frontier_settings = crawler.settings.get('FRONTERA_SETTINGS', None)
-        if not frontier_settings:
+        frontera_settings = crawler.settings.get('FRONTERA_SETTINGS', None)
```

Comment:
IMO, it makes sense to move these 3 lines into ``SettingsWrapper``, and access ``FRONTERA_SETTINGS`` variable from there, removing first argument to ``SettingsWrapper``.